frontend/packages/wizard/src/wizard.tsx

```diff
@@ -255,11 +268,16 @@ const Wizard = ({
           marginTop={0}
         >
           <StepperContainer item xs="auto" $orientation={orientation}>
-            <Stepper activeStep={state.activeStep} orientation={orientation}>
+            <Stepper
+              activeStep={state.activeStep}
+              orientation={orientation}
+              nonLinear={nonLinear}
+              handleStepClick={handleStepClick}
+            >
               {filteredChildren.map((child: WizardChildren) => {
-                const { name } = child.props;
+                const { name, isComplete } = child.props;
```

Comment:
Sorry, didn't notice this until I manually went through the code.

We have the `isComplete` prop on the Step itself but we'll need to support the prop on the WizardStep as well so it's passing it down correctly. Right now the `child.props` might have it or it might not. Should be able to mimic the way we handle isLoading/error on the WizardStep.